docker-compose/docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3.7'
-
```

Comment:
https://docs.docker.com/reference/compose-file/version-and-name/#version-top-level-element-obsolete